src/Integration.Vsix/CFamily/CFamilyStrings.resx

```diff
@@ -195,4 +195,7 @@
   <data name="ERROR_InvalidCompilationEntry" xml:space="preserve">
     <value>Invalid compilation database entry. Either Command or Arguments must be supplied, but not both.</value>
   </data>
+  <data name="ERROR_CreatingVcxRequest" xml:space="preserve">
```

Comment:
There is a missing dot here; I fixed it and will push along with the rest of the review comments.